blockchain/indexers/indexers_test.go

```diff
@@ -565,7 +565,7 @@ func TestProveUtxos(t *testing.T) {
 		for _, spendable := range spendables {
 			utxo, err := chain.FetchUtxoEntry(spendable.PrevOut)
 			if err != nil {
-				t.Fatal(fmt.Sprintf("TestProveUtxos fail. err: outpoint %s not found.",
+				t.Fatalf(fmt.Sprintf("TestProveUtxos fail. err: outpoint %s not found.",
```

Comment:
```
blockchain/indexers/indexers_test.go:568:5: S1038: should use t.Fatalf(...) instead of t.Fatal(fmt.Sprintf(...)) (gosimple)
                                t.Fatal(fmt.Sprintf("TestProveUtxos fail. err: outpoint %s not found.",
                                ^
blockchain/indexers/indexers_test.go:584:6: S1038: should use t.Fatalf(...) instead of t.Fatal(fmt.Sprintf(...)) (gosimple)
                                        t.Fatal(fmt.Sprintf("TestProveUtxos fail."+
                                        ^
blockchain/indexers/indexers_test.go:591:6: S1038: should use t.Fatalf(...) instead of t.Fatal(fmt.Sprintf(...)) (gosimple)
                                        t.Fatal(fmt.Sprintf("TestProveUtxos fail."+
```

Govet suggests using `t.Fatalf`instead of `t.Fatal(fmt.Sprintf())`.

So these lines should be 
`t.Fatalf("TestProveUtxos fail. err: outpoint %s not found.", `

Reply:
Okay, I will make these changes